website/content/labs/lab4.md

```diff
@@ -106,35 +105,36 @@ Runtime errors make the program crash while it is running, and sometimes they ar
 
 `frame n` (where `n` is the frame number from the backtrace) allows you to go to the function in that frame and see what line in that particular function caused the error.
 
-Compile `task3.c` and run it. You will segfault. Now it’s time to debug! There are three errors in this file. For each error, answer the following questions in `task3.txt`. It is a good idea to fix one error before moving onto the next.
+Compile `task3.c` and run it. You will segfault. Now it’s time to debug! There are three errors in this file. For each error, answer the following questions in `task3.txt`. It is a good idea to fix one error before moving on to the next.
 
-1. Where was the segfault? Give the function, line number, and the line of code itself.
-2. What variables were involved?
-3. Why was this causing an error?
-4. What gdb commands did you use to debug this, and what did they help you with?
-5. Fix the errors in `task3.c` (do this in the C file directly)
+For each error, answer the following question on Gradescope:
 
+* Where was the segfault? Give the function, line number, and copy-paste the line of code itself. What was causing the segfault and how did you fix it?
+
+Make sure to also commit and push your fix for each segfault!
+
+You must also answer the following question for the entire task:
+
+* Describe the steps you took in gdb to narrow down the exact location of a segfault. You can also copy-paste the contents of one debugging session with gdb, with a brief note at the end explaining how that debugging session narrowed down the location of the error.
```

Comment:
Can we change this to "Describe the steps you took in gdb to narrow down the exact location of ~a~ **one of the** segfaults"? The first time or two I read it, it seemed like we wanted this for every segfault

Reply:
Fixed.